### DISCUSSION TITLE
Issue with RefinableString and LocalPeopleResults

### DISCUSSION THREAD

**Discussion by IainMac81** [microsoft-search/pnp-modern-search#2546] [Q&A] [answered]
Hi

I have created a few new Properties in 'User Profiles' within the Admin center;

![image](https://user-images.githubusercontent.com/118296021/201978084-6ce9d87f-95dc-430a-beb4-e8b4a84a6316.png)

And assigned them to RefinableStrings in Managed Properties

![image](https://user-images.githubusercontent.com/118296021/201978352-200f3c91-c5a9-4a17-a547-ca51481081e1.png)

I can select the OWS_ mapped crawled properties in the PnP search results web part in **Selected properties** but none of the refinablestrigs relating to People: (103 and 109)

![image](https://user-images.githubusercontent.com/118296021/201980088-5e9fe807-f2a7-437e-9731-3ee51c7bbeeb.png)

- they have been setup over 24 hours
- I can use RefinableString103 as a filter
- using version 4.6.1.0

I just can not select in the WP selected properties.

Can anyone help?

Thanks

**Comment by wobba** [comment, marked as answer]
Put the cursor in the "Selected properties" box. Type the name of the managed property you want to use. Hit ENTER. It should now be in the list selected.

**Reply by IainMac81** [reply]
Great this worked thank you for this!, I needed to be careful as I have removed all other properties a few times!

**Reply by IainMac81** [reply]
Although I can now add the refinablestrings as properties, they are still not showing, even after 24 hours. Any ideas?

![image](https://user-images.githubusercontent.com/118296021/202666196-e3bf4aa7-42b6-431c-b9c6-7e85e19a3441.png)

![image](https://user-images.githubusercontent.com/118296021/202666250-0e7b4a4f-ab98-4ff3-abd0-f0954dcf2dcf.png)

Is there a way to reindex the SPO LocalPeopleresuts? 

I found how to reindex lists and libraries but not the LocalPeopleResults

Thanks

Iain

**Reply by wobba** [reply]
https://pnp.github.io/script-samples/spo-request-pnp-reindex-user-profile/README.html :)

**Reply by IainMac81** [reply]
Thanks for this link, ran it but made no difference. I managed to find a workaround by filtering with the query template tio filter on emails and remove all profiles that have been deactivated (and marked as hide from address lists in AD)

{searchTerms} * UserName:*[domain].com
-"SPS-HideFromAddressLists":1

**Reply by wobba** [reply]
After you run it it can take up to 24-48h before data is available - assuming you mapped the correct crawled properties.
